Title: Support for Longer Passwords
Author: Eric Pruitt (https://www.codevat.com/, https://github.com/ericpruitt/)

This patch enlarges the buffer used to store user input so slock can support
longer passwords.

This patch depends on "slock-00-pam-authentication.diff".

diff --git slock.c slock.c
index d55eb3d..e31e489 100644
--- slock.c
+++ slock.c
@@ -224 +224 @@ readpw(Display *dpy, struct xrandr *rr, struct lock **locks, int nscreens,
-	char buf[32], *inputhash;
+	char buf[1024], *inputhash;
